sympy/logic/boolalg.py

```diff
@@ -662,7 +680,7 @@ def _eval_simplify(self, **kwargs):
                     if (e.lhs != x or x in e.rhs.free_symbols) and x not in reps:
                         try:
                             m, b = linear_coeffs(
-                                Add(e.lhs, -e.rhs, evaluate=False), x)
+                                e.rewrite(Add, evaluate=False), x)
```

Comment:
This seems to be reverting changes that were recently made in master but that are unrelated to the PR.

Reply:
Sorry. That was my carelessness. New to open sourcing.